app/views/components/docs/content-metadata.yml

```diff
@@ -0,0 +1,34 @@
+name: Content Metadata
+description: Lists publication dates and publishers.
+shared_accessibility_criteria:
+- link
+
+examples:
+  default:
+    data:
+      from:
+        - <a href='/government/organisations/ministry-of-defence'>Ministry of Defence</a>
+      published: "31 July 2017"
+      last_updated: "20 September 2017"
+  with_history_link:
+    data:
+      from:
+        - <a href='/government/organisations/ministry-of-defence'>Ministry of Defence</a>
+      published: "31 July 2017"
+      last_updated: "20 September 2017"
+      link_to_history: true
+  two_publishers:
+    data:
+      from:
+        - <a href='/government/organisations/department-for-education'>Department for Education</a>,
```

Comment:
There's some trailing commas at the end of a few lines, these need to be removed. Otherwise commas show up i n the examples.

![screen shot 2017-11-14 at 11 07 50](https://user-images.githubusercontent.com/319055/32776911-134bd0ec-c92c-11e7-87fa-6a76bc7becd1.png)

Reply:
![screen shot 2017-11-14 at 11 08 33](https://user-images.githubusercontent.com/319055/32776939-2e870e76-c92c-11e7-9f2f-17ebc3b5a7f0.png)